pyfolio/tests/test_timeseries.py

```diff
@@ -289,7 +289,7 @@ def test_sharpe_2(self, returns, rolling_sharpe_window, expected):
             returns, rolling_sharpe_window).values.tolist()), expected)
 
     @parameterized.expand([
-        (simple_rets, 0.010766923838471554)
+        (simple_rets, 0.10376378866671222)
```

Comment:
Reworded that commit: https://github.com/quantopian/pyfolio/commit/ca8567e2ac2d9b1cae1c4242fdf571689d1416c8